src/coreclr/debug/di/module.cpp

```diff
@@ -5169,7 +5169,7 @@ HRESULT CordbNativeCode::GetReturnValueLiveOffsetImpl(Instantiation *currentInst
 
                 int skipBytes = 0;
 
-#if defined(TARGET_X86) && defined(FEATURE_CORESYSTEM)
+#if defined(TARGET_X86)
```

Comment:
This is support for `PSEUDORANDOM_NOP_INSERTION` that is disabled in the JIT and there is an issue to delete it completely. We may want to change this ifdef to `#ifdef PSEUDORANDOM_NOP_INSERTION`